### PR TITLE
login inputs + logo

### PR DESCRIPTION
**Pull request by squashgray**
The most beautiful login page that ever was.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import logo from './logo.svg';
-import './App.css';
+
```
Comment (zimashima) [inline src/App.tsx:1]:
```suggestion
```

**Review by jae-finger — APPROVED** [review]
:]

**Review by allie-rae — APPROVED** [review]
cuuuute omg

**Review by danimal-johnson — APPROVED** [review]
Thanks for the Logos, Mashima!
Thanks for the Material UI tutorial, Alliie!

**Review by zimashima — APPROVED** [review]
What a cute logo! Must have been created by someone cool xD

File: src/App.tsx [hunk 2]
```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
 import AppRoutes from './routes/AppRoutes';
 import { useGetDogByIdQuery } from './redux/dog/createApi';
 
 function App() {
   const { data } = useGetDogByIdQuery(1);
   console.log('data', data);
```
Comment (zimashima) [inline src/App.tsx:6]:
```suggestion
```

**Review by allie-rae — APPROVED** [review]

**Pull request merged**